src/redux/reducers/userReducer.ts

```diff
@@ -24,18 +27,24 @@ export const userReducer = (state: User = emptyUser, action: UserActions | Orcid
         .map(affiliationString => affiliationString.trim())
         .filter(affiliation => affiliation) as Affiliation[];
       const roles = action.user['custom:applicationRoles'].split(',') as RoleName[];
+      const isLoggedIn = true;
+      const email = action.user.email;
       const user: Partial<User> = {
         name: action.user.name,
-        email: action.user.email,
+        email,
         id: action.user['custom:feideId'],
         institution: action.user['custom:orgName'],
         roles,
         application: action.user['custom:application'] as ApplicationName,
-        isLoggedIn: true,
+        isLoggedIn,
         organizationId: getOrganizationIdByOrganizationNumber(action.user['custom:orgNumber']),
         affiliations,
         givenName: action.user.given_name,
         familyName: action.user.family_name,
+        isPublisher: affiliations.some(userAffiliation => publisherAffiliations.includes(userAffiliation)),
+        isAppAdmin: email.endsWith('@unit.no'),
```

Comment:
denne er vel ikke helt kosher 😬 men blir kanskje erstattet av noe cognito-greier etterhvert eller?

Reply:
Hele opplegget rundt roller er et stort spørsmålstegn for meg.

Reply:
Har avklart med backend og vi ble enig om alt skal være roller